op-acceptor/cmd/main.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 var (
-	Version   = "v0.1.0"
+	Version   = "v0.5.0"
```

Comment:
we don't typically bother and I don't think it matters. But I noticed it, so I bumped it. 0.5.0 will be the next release as these are potentially breaking changes